rllib/agents/trainer.py

```diff
@@ -40,13 +40,87 @@
 # yapf: disable
 # __sphinx_doc_begin__
 COMMON_CONFIG = {
-    # === Debugging ===
-    # Whether to write episode stats and videos to the agent log dir
+    # === Settings for Rollout Worker processes ===
+    # Number of rollout worker actors to create for parallel sampling. Setting
+    # this to 0 will force rollouts to be done in the trainer actor.
+    "num_workers": 2,
+    # Number of environments to evaluate vectorwise per worker. This enables
+    # model inference batching, which can improve performance a lot if
+    # inference is the bottleneck for rollouts.
+    "num_envs_per_worker": 1,
```

Comment:
Looks good - I would also mention that this introduces a source of randomness iirc